apps/backoffice-v2/src/pages/Entity/components/TableCell/interfaces.ts

```diff
@@ -0,0 +1,33 @@
+import { ComponentProps } from 'react';
+import {
+  Table,
+  TableBody,
+  TableCaption,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '../../../../common/components/atoms/Table';
+import { ColumnDef, RowData, TableOptions } from '@tanstack/react-table';
+
+export interface ITableCellProps<TData extends RowData, TValue = unknown> {
+  // Props to be used explicitly by the cell inside 'value'. Props outside 'value' to be used by the blocks API.
```

Comment:
Do you feel like the comments in this file are necessary for the understanding of the interface?

Reply:
My thought process here is wondering how this interface would look like to someone who is unfamiliar with blocks and their concepts. Let me know if they should still be removed with that in mind.

Reply:
I think this type of instructions could go really well into a `README.md` file as they are relevant for a lot of our UI components, making a good reference for new developers